e2e/test/scenarios/collections/collection-pinned-overview.cy.spec.js

```diff
@@ -241,9 +265,244 @@ describe("scenarios > collection pinned items overview", () => {
       .findByText("Orders, Count, Grouped by Created At (year)")
       .should("exist");
   });
-});
 
-const openRootCollection = () => {
-  cy.visit("/collection/root");
-  cy.wait("@getPinnedItems");
-};
+  describe("scenarios > collection pinned items overview > pinned model description tooltip", () => {
```

Comment:
There are no backend calls or browser-specific logic (e.g. something rendering or layout-specific) that we test here, so I think we could (should?) test it in Jest in tests for the affected components.

Reply:
There is a **draft** doc that clarifies the part about e2e. It's probably the best I have right now https://www.notion.so/metabase/Shift-the-engineering-quality-strategy-c75e948910654f348ec345c20b7044f6?pvs=4#ca158621ff3a4f92a9d4384fa053e1de

Reply:
In my own words, I'd say that we add e2e tests for:
- any new shape or data coming from the BE
- any flows that involve interacting with the BE **and** processing the response, e.g. updating an entity or something similar
- cross-feature functionality, e.g. something that isn't scoped to a particular area and would require rendering the whole app